apps/etterlatte-saksbehandling-ui/client/src/components/behandling/trygdetid/detaljer/TrygdetidDetaljer.tsx

```diff
@@ -31,16 +31,24 @@ export const TrygdetidDetaljer: React.FC<Props> = ({ beregnetTrygdetid }) => {
   )
 }
 
-export const formaterBeregnetTrygdetid = (beregnetTT: IBeregnetTrygdetidAarMaaneder) => {
-  return `${beregnetTT.aar} år${beregnetTT.maaneder ? ` ${beregnetTT.maaneder} måneder` : ''}`
+export const formaterBeregnetTrygdetid = (periode?: string) => {
+  if (periode) {
+    const aar = periode.substring(1, periode.indexOf('Y'))
+    const maaneder = periode.includes('M') ? periode.substring(periode.indexOf('Y') + 1, periode.indexOf('M')) : null
+
+    return `${aar} år${maaneder ? ` ${maaneder} måneder` : ''}`
+  }
+  return ''
```

Comment:
Brukte litt tid på å parse koden her -- forslaget under passer bedre med min tankegang, men du får bestemme om du er enig :p 

```suggestion
export const formaterBeregnetTrygdetid = (periode?: string) => {
  if (!periode) {
    return ''
  }
  // formatet på periode matcher /\d+Y(\d+M)?/, så en split på Y|M vil gi en array med
  // 1. år-strengen og 2. tom streng eller måned-strengen
  const [aar, maaneder] = periode.split(/Y|M/)
  return `${aar} år${maaneder ? ` ${maaneder} måneder` : ''}`
```

Reply:
I utgangspunktet enig, men det viser seg å være litt mer kompliserende av at man ikke alltid får `Y`.
Dvs, perioden ser. vanligvis slik ut: `P20Y10M`, men dersom det ikke finnes noen år, så blir den sånn: `P20Y`, og dersom det ikke finnes noen måneder, så blir den sånn: `P10M`. Har du noe regex-magi som håndterer det? 😅

Reply:
Man kan evt. sørge for at man alltid får antall år, men da blir den ikke fullt så smooth.
```typescript
export const formaterBeregnetTrygdetid = (periode?: string) => {
  if (!periode) {
    return ''
  }

  // Legger til 0 år eksplisitt dersom perioden er under ett år
  const periodeMedAntallAar = periode.includes('Y') ? periode : 'P0Y' + periode.slice(1)

  // formatet på periode matcher /\d+Y(\d+M)?/, så en split på Y|M vil gi en array med
  // 1. år-strengen og 2. tom streng eller måned-strengen
  const [aar, maaneder] = periodeMedAntallAar.slice(1).split(/Y|M/)
  return `${aar} år${maaneder ? ` ${maaneder} måneder` : ''}`
}
```